infrastructure/load-balancers.yaml

```diff
@@ -43,6 +57,20 @@ Resources:
                 - Type: forward
                   TargetGroupArn: !Ref DefaultTargetGroup
 
+    LoadBalancerListenerTls:
+        Condition: SslCertificate
+        Type: AWS::ElasticLoadBalancingV2::Listener
+        Properties:
+            Certificates:
+                # - CertificateArn: !Ref PublicAlbAcmCertificate
+                - CertificateArn: arn:aws:acm:us-west-2:845828040396:certificate/22d1eab8-34cb-48ed-b404-91a3ecaeed08
```

Comment:
I'm assuming this is temporary as you try and get this working, but an important unforeseen consequence is occurring:  the Certificate itself isn't depending on `PublicAlbAcmCertificate`, but the conditions above still are.